cmd/termui/presenter/presenterStatusHandler.go

```diff
@@ -11,7 +11,8 @@ var maxLogLines = 100
 
 // PresenterStatusHandler is the AppStatusHandler impl that is able to process and store received data
 type PresenterStatusHandler struct {
-	presenterMetrics            *sync.Map
+	presenterMetrics            map[interface{}]interface{}
```

Comment:
I think the key can be a string. because all metrics consts are strings.

Reply:
done